spec/game_spec.rb

```diff
@@ -6,7 +6,8 @@
     stdin = StringIO.new
     stdout = StringIO.new
     game = Game.new(input: stdin, output: stdout)
-    allow(stdin).to receive(:gets).and_return("R2", "3PO", "C1")
+    allow(stdin).to receive(:gets)
```

Comment:
This test has been growing over time and the title doesn't make a lot of sense anymore. 

I would love to see multiple tests, testing different acceptance level important steps. That means that 

1. You need to start parsing the output and understanding which way part of the printing relates to which functionality
2. Find a way to inject in some way the grid data.

Reply:
Agreed. I'll do this in a follow-up PR.